plugins/woocommerce/i18n/states.php

```diff
@@ -322,7 +322,7 @@
 		'CN32' => __( 'Xinjiang / 新疆', 'woocommerce' ),
 	),
 	'CO' => array( // Colombia States.
-		'CO-DC' => __( 'Distrito Capital de Bogotá', 'woocommerce' ),
+		'CO-DC' => __( 'Capital District', 'woocommerce' ),
```

Comment:
Can you place this under Cundinamarca per the [CLDR](https://github.com/unicode-org/cldr/blob/release-40/common/subdivisions/en.xml#L894)? Ty!